pkg/route/ingress/ingress.go

```diff
@@ -370,6 +372,27 @@ func (c *Controller) sync(key queueKey) error {
 		return err
 	}
 
+	// If the ingress specifies an ingressclass and the ingressclass does
+	// not specify OpenShift Router as its controller, ignore the ingress.
```

Comment:
We should ignore ingresses that do not specify `openshift.io/ingress-to-route` as their controller, right? (`OpenShift Router` seems a little vague, but perhaps thats intentional?)

Reply:
Right.  An earlier version of these changes used the controller name "openshift.io/haproxy-router", which I changed to "openshift.io/ingress-to-route" (since it's really the ingress-to-route controller that is managing the ingresses), and I forgot to update the comment when I changed the controller name.  Thanks for catching that!